cmd/devdata/system_intake_decision_actions.go

```diff
@@ -22,18 +22,32 @@ func makeSystemIntakeAndIssueLCID(
 	store *storage.Store,
 ) *models.SystemIntake {
 	pastMeetingDate := time.Now().AddDate(0, -1, 0)
-	intake := makeSystemIntakeAndProgressToStep(
-		requestName,
-		intakeID,
-		requesterEUA,
-		logger,
-		store,
-		model.SystemIntakeStepToProgressToGrbMeeting,
-		&progressOptions{
-			completeOtherSteps: true,
-			meetingDate:        &pastMeetingDate,
-		},
-	)
+	var intake *models.SystemIntake
+	// business cases require EUA ID to be set in DB constraint
```

Comment:
I don't think I get this comment. If EUA IDs are required in the DB constraint for a business case, why are we choosing to progress to the Draft Business Case step when there is no EUA ID passed?

Reply:
Progressing to the draft business case doesn't actually create the business case, it just creates the intake, fills and submits the initial form, and progresses to the draft biz case step.

Reply:
Then what's the relevance of this comment? I don't think it quite makes sense to me why we can't just always progress to one step or the other here instead of basing it on the presence of `requesterEUA` being passed in here.

Reply:
There's an option in the function parameters to complete prior steps, and in normal scenarios it fills out all fields on the intake and completes all previous steps, however if the EUA isn't given the DB constraint causes the code to fail.

This error popped up when I was making intakes with LCIDs issued and using the function to generate completely filled out intakes that had gone through all steps in the process. The if statement here is just covering that missing EUA edge case and progressing only to the step before a business case is created.